README.md

```diff
@@ -42,7 +42,24 @@ All features are available for it.
 
 ## Installation and updating
 
-Installation is pretty much the same as upstream. \
-Please read the [glitch-soc docs](https://glitch-soc.github.io/docs/) for instructions.
+### Installing from scratch
 
-Update steps are also the same, but for convenience an update script is available on the [scripts repo](https://github.com/polyamspace/scripts)
+Installation is the same as described in the [Mastodon documentation](https://docs.joinmastodon.org/admin/install/) \
+Replace the repo URl with this one.
+
+### Switching to Polyam-glitch
+
+It is possible to switch from vanilla and glitch-soc to polyam-glitch the same way as switching from vanilla to glitch-soc. \
+Please read the [glitch-soc docs](https://glitch-soc.github.io/docs/) for instructions. \
+Replace the glitch-soc repo with this one. \
+Alternatively, the [update script](https://github.com/polyamspace/scripts/blob/main/update.sh) can be used, \
+though it is primarily intended for updating existing polyam-glitch installations.
+
+Switching from other forks might require some manual adjustments, especially with duplicate migrations.
```

Comment:
I'm not sure this is that much of a concern anymore as every polyam migration should be skipped on conflict.
Although I have to double check that. (Reactions might have some conflicting migrations)

Reply:
Indeed. The original reaction migration was changed.
So currently, when running glitch-soc with the reaction patch and switching to polyam-glitch, migrating the DB should fail.